app/controllers/users_controller.rb

```diff
@@ -65,6 +65,8 @@ def create
   def create_external
     @user = User.new(params[:user])
     @user.password = generate_new_password
+    @user.created_by = current_user.try(:id)
```

Comment:
I think I'd maybe leave this right now for consistency and then change all of them together?

That said, I don't really need the `created_by` anymore, we can just ditch it.